### PR TITLE
Toggle events action dropdown on link click in the event page

### PR DESCRIPTION
**Pull request by arun1595**
#### Motivation and Context
Fixes https://github.com/AgileVentures/WebsiteOne/issues/1711

#### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

#### Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by tansaku** [comment]
thanks @arun1595 do you think it would make sense to have a js unit test for this?

File: config/database.yml [hunk 1]
```diff
@@ -3,8 +3,8 @@ development: &dev
   encoding: unicode
   database: websiteone_development
   pool: 20
-  username: postgres
-  password:
+  username: lazyass
```
Comment (tansaku) [inline config/database.yml]: I think you want to avoid checking this in :-)

**Comment by tansaku** [comment]
@arun1595 I got your sample test running by  adding `_spec` to it:

**Comment by tansaku** [comment]
I removed the `beforeEach` function from your non-appearing test and restarted `rake jasmine` and I get the test showing up and failing.  I suspect there is some sort of issue in the `beforeEach` that's failing badly and screwing with the test framework ... maybe

File: config/database.yml (hunk 1 above)
Comment (arun1595) [inline config/database.yml]: My bad, I thought I had checked this file out before committing.

**Comment by arun1595** [comment]
@tansaku The issue has been fixed, PTAL :smile:

**Comment by tansaku** [comment]
great work @arun1595 - many thanks for this :-)

**Pull request merged**